src/www/ServerSocket.cpp

```diff
@@ -67,11 +69,14 @@ bool ServerSocket::create(void) {
   int opt = 1;
 
   m_fd = socket(AF_INET, SOCK_STREAM, 0);
+  std::cout << "ServerSocket Made" << std::endl;
   if (m_fd == -1) return false;
   if (setsockopt(m_fd, SOL_SOCKET, SO_REUSEADDR, &opt, sizeof(opt)) < 0) {
+    std::cout << "ServerSocket options fucked" << std::endl;
     this->close();
     return false;
   }
+  std::cout << "ServerSocket should be good to go" << std::endl;
```

Comment:
Nobody wants to use my logger😔

Reply:
I don't know how the logger works :')
Teach me senpai